commands.go

```diff
@@ -33,9 +32,6 @@ func init() {
 		"acl-controller": func() (cli.Command, error) {
 			return &cmdController.Command{UI: ui}, nil
 		},
-		"health-sync": func() (cli.Command, error) {
-			return &cmdHealthSync.Command{UI: ui}, nil
-		},
```

Comment:
nit: This isn't deprecating the `health-sync` command, it is removing it completely. I think this is a breaking change so we should ensure that the changelog gets updated accordingly.

Reply:
fwiw, this is merging into a non-main / feature branch, so we can either add the changelog entry now or when this feature branch goes into main. I'm fine either way.

But agreed! There are many breaking changes for agentless that need to be highlighted in the changelog, and it would probably be good to start adding those to the changelog on the feature branch to keep track of them.

Reply:
The whole of agentless is a breaking change to Consul ECS. I am planning to accumulate all the changes from all the PRs raised against the feature branch and add them in the final PR that gets merged to the main branch.